.storybook/decorators/storyTemplates.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+export const defaultTemplate = (config) => (
+  (story) => (
+    <div style={{padding: '0 20px'}}>
+      <header className="page-header">
+        <h2>{config.title}</h2>
+      </header>
+      {!config.documentationLink &&
+      <p>
+        This pattern does not yet exist in <a href="http://www.patternfly.org/pattern-library/">PatternFly</a>.
+      </p>
+      }
+      {(config.description && !config.documentationLink) &&
+      <p>{config.description}</p>
```

Comment:
It may be better to make this less strict. The description can be provided to describe React specific usage or functionality of that component. This information is not provided in patternfly docs. So I'd prefer displaying description always if it is provided. Also, by not putting description into <p>, allows for the description to be any JSX, such as code example etc. not just single text paragraph.